algos.yaml

```diff
@@ -714,6 +753,27 @@ msspacev-1B:
                 {"nProbe":170, "efSearch":510, "K1":128, "blockSize":4096, "identifier": "bbann"},
                 {"nProbe":180, "efSearch":360, "K1":128, "blockSize":4096, "identifier": "bbann"}
               ]
+=======
```

Comment:
Clean up markers from merge conflicts and verify the contents?

Reply:
yep